aten/src/ATen/native/native_functions.yaml

```diff
@@ -1802,6 +1802,14 @@
   dispatch:
     CUDA: miopen_depthwise_convolution_backward_weight
 
+- func: hb_convolution_transpose(Tensor self, Tensor weight, Tensor? bias, int[] padding, int[] output_padding, int[] stride, int[] dilation, int groups) -> Tensor
```

Comment:
> It seems to me that we can directly call `at::native::hb_convolution_transpose` instead of going through dynamic dispatching?

Are we preparing for having a sparse version? If no then I guess we don't need dispatching here, since only HammerBlade calls this?

Reply:
This is some thing I don't understand fully. I tried to follow cudnn here:
https://github.com/cornell-brg/hb-pytorch/blob/cf58f27867492f137d97aaec547f33c7f54540c3/aten/src/ATen/native/native_functions.yaml#L790-L794

And all other calls in this file are made to `at::` namespace, so didn't choose to directly invoke the `at::native` here...

Reply:
emmm... I looked at git blame and it shows Edward Z. Yang added this ... he must have some reason for doing so ... let's follow the cudnn style then

Reply:
I just verified directly calling ` at::native::hb_convolution_transpose` works. 

But no call in that file is made to `at::native::` namespace. What do you think? Would you like me to change it?

Reply:
Personally I think it would be nice to stay the same as cuDNN ;)